cargo-clone-core/src/lib.rs

```diff
@@ -159,11 +160,19 @@ fn select_pkg<'a, T>(
 where
     T: Source + 'a,
 {
-    src.update()?;
+    src.invalidate_cache();
```

Comment:
[This](https://github.com/rust-lang/cargo/commit/f12f0256caedaeada2fecd8bc5e93feee7abb5b7) is the commit where the `update` function was removed.
The git commit comment is quite good 👍 
A possible optimisation could be to call the `invalidate_cache` function only once. I think at the moment we call it for every crate we download. What do you think? We could open an issue and implement this in a separate PR.

Reply:
Yes, let's open a ticket for that.

Reply:
#56